website/docs/segments/pnpm.mdx

```diff
@@ -15,9 +15,8 @@ import Config from "@site/src/components/Config.js";
 <Config
   data={{
     type: "pnpm",
-    style: "diamond",
-    leading_diamond: "\uE0B2",
-    trailing_diamond: "\uE0D6",
+    style: "powerline",
```

Comment:
I don't see why this is necessary, can you elaborate?

Reply:
1. Most docs use the powerline style
2. The symbols/diamonds used were for the right side prompt

Feel free to revert that change before merging, I was just trying realign for consistency